packages/dd-trace/src/appsec/waf/waf_context_wrapper.js

```diff
@@ -51,6 +51,11 @@ class WAFContextWrapper {
 
     if (!payloadHasData) return
 
+    if (this.ddwafContext.disposed) {
+      log.warn('Calling run on a disposed context')
+      return
+    }
```

Comment:
worth putting this check at the start of the method so we don't waste time parsing data in case it's already disposed

Reply:
Moved the check to the start of the method.